### PR TITLE
fix: support list of [x,y] for creating polygon gates

### PR DESCRIPTION
**Pull request by gegnew**
Specifically, now `createPolygonGate` supports a single `vertices` arg, which is a list of vectors: `list(c(x,y), c(x,y), ...)`

After PR, rebuild documentation.

Closes https://github.com/primitybio/cellengine-r-toolkit/issues/39

### CODE REVIEW

**Review by zbjornson — COMMENTED** [review]
For now, to avoid a breaking change, can you continue to support `xVertices` and `yVertices` in addition to `vertices`, but mark them deprecated in the doc string please?

Don't need tests for `xVertices`+`yVertices` anymore though; you can leave the test changes as-is.

**Comment by gegnew** [comment]
:partly_sunny:

File: R/createPolygonGate.R [hunk 1]
```diff
@@ -6,8 +6,11 @@
 #' @param xChannel The name of the x channel to which the gate applies.
 #' @param yChannel The name of the y channel to which the gate applies.
 #' @param name The name of the gate.
-#' @param xVertices List of x coordinates for the polygon's vertices.
-#' @param yVertices List of y coordinates for the polygon's vertices.
+#' @param vertices List of vectors of coordinates, like [c(x,y), c(x,y), ...]
```
Comment (zbjornson) [inline R/createPolygonGate.R:9]: Should this say `like list(c(x,y), c(x,y), ...)` instead of square brackets?

**Comment by zbjornson** [comment]
This just means we need a doc rebuild, right?

```
Codoc mismatches from documentation object 'createPolygonGate':
createPolygonGate
  Code: function(experimentId, xChannel, yChannel, name, vertices =
                 list(), xVertices = c(), yVertices = c(), label =
                 NULL, gid = generateId(), parentPopulationId = NULL,
                 parentPopulation = NULL, tailoredPerFile = FALSE,
                 fcsFileId = NULL, fcsFile = NULL, locked = FALSE,
                 createPopulation = TRUE)
  Docs: function(experimentId, xChannel, yChannel, name, xVertices =
                 c(), yVertices = c(), label = c(mean(xVertices),
                 mean(yVertices)), gid = generateId(),
                 parentPopulationId = NULL, parentPopulation = NULL,
                 tailoredPerFile = FALSE, fcsFileId = NULL, fcsFile =
                 NULL, locked = FALSE, createPopulation = TRUE)
  Argument names in code not in docs:
    vertices
  Mismatches in argument names (first 3):
    Position: 5 Code: vertices Docs: xVertices
    Position: 6 Code: xVertices Docs: yVertices
    Position: 7 Code: yVertices Docs: label
  Mismatches in argument default values:
    Name: 'label' Code: NULL Docs: c(mean(xVertices), mean(yVertices))
```

**Comment by zbjornson** [comment]
I think the RDoc files need to be rebuilt (not just the HTML files).

They're stale on `master` also, so we could merge this and do a rebuild later.

**Comment by gegnew** [comment]
Hmm, yeah, I think I've gone this whole time not realizing that roxygen does not rebuild the Rdoc if the file exists. I'll rebuild on master too, after we merge this

**Review by zbjornson — APPROVED** [review]

**Pull request merged**